powerauth-java-server/pom.xml

```diff
@@ -117,6 +117,11 @@
             <artifactId>commons-text</artifactId>
             <version>${commons-text.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I was thinking about adding this dependency too. But for OIDC activation, I am trying to handle it by `NimbusJwtDecoder` from Spring Security.

Reply:
We have this in several projects already. 🤔